sound/soc/sof/fw-file-profile.c

```diff
@@ -94,37 +201,122 @@ sof_file_profile_for_ipc_type(struct device *dev,
 	return ret;
 }
 
-static void sof_print_profile_info(struct device *dev,
+static void
+sof_missing_firmware_notification(struct snd_sof_dev *sdev,
```

Comment:
nit-pick: usually "firmware notification" means some sort of firmware-generated IPC is received by the host driver. This is not what this function does. I think this is a user notification, but that's confusing.

Reply:
OK, I will find a better name for the function.

Reply:
`sof_log_missing_firmware_file()`